### ISSUE TITLE
GetRows returns incorrectly rounded values

### ISSUE THREAD

**Issue by daenmdir** [qax-os/excelize#1410]
**Description**
`GetRows` with option `RawCellValue` set to `false` floors numeric values instead of rounding them per cell formatting rules. 

Please note that this issue only occurs for me with the files that my colleagues create using MS Excel 2019.
It doesn't reproduce when I create or re-save the same file without any changes with my LibreOffice.

**Steps to reproduce the issue:**
[round.xlsx](https://github.com/qax-os/excelize/files/10131714/round.xlsx) has 1.3, 1.5, 1.7 values for cells A1 through C1 with numeric formatting and 0 decimals so excel/libre rounds them like this:
![image](https://user-images.githubusercontent.com/119590391/205036764-c1256819-d627-4b1f-b1a6-edfdc3264e5d.png)

Here's a piece of code displaying the problem:
```go
f, _ := excelize.OpenFile("round.xlsx")
defer f.Close()

rv, _ := f.GetRows("Лист1")
fmt.Println(rv)  // [[1 1 1]] floored instead of being rounded

rv, _ = f.GetRows("Лист1", excelize.Options{
	RawCellValue: true,
})
fmt.Println(rv) // [[1.3 1.5 1.7]] correct raw values
```

**Describe the results you received:**
First print output is `[[1 1 1]]`
**Describe the results you expected:**
First print output is `[[1 2 2]]`

**Output of `go version`:**
```text
go version go1.18.3 linux/amd64
```

**Excelize version or commit ID:**
```text
v2.6.1
```

**Comment by xuri** [comment]
Thanks for your issue, this issue has been fixed, please upgrade to the master branch code. This fix will be released in the next version [2.7.0](https://xuri.me/excelize/ru/releases/v2.7.0.html)

**Comment by xuri** [comment]
I closed this issue. Please let me know if you have any questions, and reopen this anytime.
